staff/aleix-caminal/cv/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+    <head>
+        <meta charset="utf-8">
+        <title>Curriculum Vitae</title>
+    </head>
+    <body>
+        <main>
+            <section id="head">
+                <div class="image">
+                    <img src="assets/img/profile.jpg" alt="profile">
+                </div>
+                <div class="info">
+                    <h4 class="name">Charles Tucker III</h4>
+                    <p class="web">Web: <a href="http://memory-alpha.wikia.com/wiki/Charles_Tucker_III">http://memory-alpha.wikia.com/wiki/Charles_Tucker_III</a></p>
+                    <p class="email">E-mail: <a href="mailto:charlestrip@starfleet.com">charlestrip@starfleet.com</a></p>
+                    <p class="mobile">Mobile: 07791 858292</p>
+                    <address>
+                        <p class="street">8 Louisiana Ave</p>
+                        <p class="town">Panama City, Florida</p>
+                        <p class="country">North America, United Earth</p>
+                    </address>
+                </div>
+            </section>
+            <section id="education">
+                <h2>Education and Qualifications</h2>
+                <article>
+                    <h3>Self Taught Engineering</h3>
+                    <time>?-2139</time>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+                </article>
+                <article>
+                    <h3>Bayshore Elementary</h3>
+                    <time>2126-2138</time>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+                </article>
+            </section>
+            <section id="experience">
+                <h2>Working Experience</h2>
+                <article>
```

Comment:
may you want to nest sections inside sections (that is, sub-sections), instead of using article for a purpose their are not aimed to. see the differences here:
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section